content/docs/command-reference/exp/remove.md

```diff
@@ -27,8 +27,15 @@ With `--queue`, the list of experiments awaiting execution is cleared instead.
 - `--queue` - remove all experiments that haven't been run yet (defined via
   `dvc exp run --queue`).
 
-- `-A`, `--all` - remove all experiments that have been run. Use `--queue` to
-  remove queued ones.
+  <admon type="warn">
+
+  `dvc exp remove --queue` is now an alias for `dvc queue remove --queued`. The
+  `--queue` flag will be deprecated in a future DVC release.
```

Comment:
@pmrowla Taking a final look through this one. Let's take this out per https://github.com/iterative/dvc.org/pull/3715/files#r921466634.